TESTING.md

```diff
@@ -328,3 +321,73 @@ imported packages: **o1, o1.llama, o1.randomtext**.
 **8.4** Double click **SwingExamples** in **Modules** list.
 
 **8.5** ASSERTION: **SwingExamples** shows up as a module in the project tree.
+
+### 9 Authentication with A+ LMS
+
+**9.1** From the main menu, choose **A+ > Set A+ Token**.
+
+![Set token](images/setToken.png)
+
+**9.2** Click on the highlighted "What's my token?" link.
+
+**9.3** ASSERTION: Browser opens your profile page in A+ LMS (login, if asked).
+
+**9.4** Copy **API Access Token** from the web page and insert it into the prompt, click "OK".
+
+**9.5** ASSERTION: The assignments tab is populated with Weekly assignments.
+
+![Assignments shown](images/assignments-shown.png)
+
+**9.6** Close the IDE.
+
+**9.7** Open the IDE with the project.
+ 
+**9.8** If [safe password storage](https://jetbrains.org/intellij/sdk/docs/basics/persisting_sensitive_data.html?search=sensitive#storage) 
+is activated in the OS.
+
+**9.9** ASSERTION: Assignments tab is still populated.
+
+### 10 Assignments
+
+#### Part I: Submitting an Assignment.
+
+**10.1** Expand **Week 1** and select **Assignment 2 (GoodStuff)**, and hit 
+**Submit A+ Assignment**.
+
+![Submit](images/submit_button.png)
+
+**10.2** ASSERTION: The "Could not find module: The A+ Courses plugin could not find the 
+module GoodStuff." warning is shown.
+
+**10.3** Go to **Modules** tab and import **GoodStuff** module like described in **6.1**
+
+**10.4** Repeat **10.1**.
+
+**10.5** ASSERTION: **Submit Assignment** dialog is shown.
+
+**10.6** Select **Submit as: > Submit alone**, click **OK**.
+
+**10.7** ASSERTION: The "Assignment successfully sent for assessment: You will be notified here when
+ feedback is available. (You may also always check any of your submissions on the course website in 
+ A+.)" notification is shown.
+ 
+#### Part II. Checking the submission.
+ 
+**10.8** Wait a while.
+
+**10.9** ASSERTION: The "Submission feedback available Feedback for Assignment 2 (Goodstuff) is now 
+available. Click the link below to open the feedback." notification is shown.
+
+**10.10** ASSERTION: Submission is added under the corresponding Assignment.
+
```

Comment:
This isn't actually true though? Unless the "Refresh Assignments" action is used

Reply:
so there's no **push**?

Reply:
By push do you mean that the submission is added to the tree? That doesn't happen at the moment.

After a refresh the new submission comes from the API and becomes visible.